smarts/sstudio/genhistories.py

```diff
@@ -1032,24 +1093,25 @@ def _check_args(args) -> bool:
         os.remove(args.output)
 
     if args.old:
-        dataset_spec = {"source": "OldJSON", "input_path": args.dataset}
+        dataset_spec = {"source_type": "OldJSON", "input_path": args.dataset}
```

Comment:
Should we be removing this old format soon? It seems it's been deprecated for a while, and now we have an "old" way of specifying the dataset spec as well as an "old" traffic history format.

Reply:
Yeah, I figure we can remove it at the same time we remove the deprecated yaml-file support (i.e., in our next major version, 0.7).